Add remaining stories

DawoodShahat opened the pull request:
Added Stories for:

- ```FilterInput.svelte``` component
- ```Pill.svelte``` component

wlach commented on stories/filterinput.stories.js:
```diff
@@ -0,0 +1,10 @@
+export const Basic = () => ({
+  Component: FilterInput,
+  props: "",
+});
```
Can you add some examples of a simple list of items being filtered? e.g.

"foo", "bar", "foobar"

DawoodShahat replied:
Alright. Adding in the next commit.

wlach reviewed (changes requested):
Thanks! I think we could flesh out the FilterInput example a bit more. Note that you can create components especially for telling stories, see e.g.

https://github.com/graph-paper-org/graph-paper/tree/main/packages/button/stories

DawoodShahat commented:
upon @wlach review
- added a list of items to be filtered by FilterInput component
- added knobs to play with both Pill and FilterInput component interactively

wlach commented:
Thanks @DawoodShahat!
